app/gitlab/lib/common.js

```diff
@@ -28,9 +28,9 @@ if (Meteor.isServer) {
 			Gitlab.configure(config);
 		}, 300);
 
-		settings.get('API_Gitlab_URL', updateConfig);
-		settings.get('Accounts_OAuth_Gitlab_identity_path', updateConfig);
-		settings.get('Accounts_OAuth_Gitlab_merge_users', updateConfig);
+		settings.watch('API_Gitlab_URL', updateConfig);
```

Comment:
Maybe a watchMultiple would be good here? 👀

Reply:
true